rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -208,6 +208,7 @@ docker run --name $docker_name \
   --network=host \
   --cpus={{{ taskData.cpuNumber }}} \
   --memory={{{ taskData.memoryMB }}}m \
+  --shm-size={{{ taskData.memoryMB }}}m \
   $nvidia_devices \
```

Comment:
What's specific problem need to solve to add this switch?

Reply:
pytorch needs to use shm when reading data